proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+2. The articles aim to:
+   1. Distribute Mixxx as a free and open source software
+   2. Distribute Manuals and Guides
+   3. Organize user support and education 
+   4. Mentor students during interneships for learning software developmen.
+   5. Maintaining the development infrstructur for easy code access
+3. Amendments to the association’s purpose may only be made in accordance with § 3 (1).
+
+## 3. Non-profit Association
+
+1. The Association’s activities are exclusively non-profit and tax-privileged in accordance with German Tax Regulations. Its activities are for the common good and not for profit-making.
+2. Financial means may only be used for purposes stipulated in the Articles of Association. Members are reimbursed for expenses but do not receive direct benefits from the Association’s funds.
+3. It is not permitted to favor someone through association expenditures that do not serve the association's purpose or are disproportionately high.
+
+## 4. Types of Membership and Members of the Association
+
+1. Membership is open to natural persons and legal bodies supporting the association's aims. Membership types include:
+   1. Active members: Natural persons actively supporting the Association’s purpose and aims.
+   2. Extraordinary members: Natural persons and legal bodies supporting the Association’s aims but without voting rights.
+   3. Supporting Members: Extraordinary members primarily supporting through financial or material contributions.
+2. Active membership is granted upon suggestion and support by active members or a majority vote. Criteria include long-term commitment and contributions.
+3. Membership terminates due to exit, expulsion, death, or loss of legal status. Written requests for termination must be submitted at least 4 weeks prior to December 31st.
+4. Expulsion may occur due to serious violations or non-payment of fees over 12 months. Members must be given the opportunity to justify their actions before expulsion.
+5. Members leaving the Association or if the Association is liquidated do not have a right to reimbursement for contributed assets.
+
+## 5. Organs of the Association
+
+- The General Assembly
+- The Board
+
+## 6. General Assembly
+
+1. The General Assembly consists of all active members, each with one vote.
+2. It takes place at least once a year, with invitations sent at least six weeks prior.
+3. An extraordinary General Assembly may be called if requested by at least 20% of members.
+4. The General Assembly has a quorum if the invitation was submitted in proper form.
+5. A ¾ majority of votes is required to amend the Articles of Association or decide the liquidation of the Association.
+6. Active members unable to attend may be represented by another member with written authorization.
+
```

Comment:
I think this is one of the key aspects that should be added. We have potential members from all over the world. As much as I would like to meet in person with all of you, I guess it's kind of unrealistic to make this happen. Hence I would be nice if follow a hybrid approach as a general rule to allow participation from all members, even if they lack the time/money/passport to travel to the meeting location.

```suggestion
7. Participation in the General Meeting must also be possible via electronic communication. By resolution of the Executive Board, a General Meeting may be held exclusively by this means.

```

I took https://www-vereinswiki-info.translate.goog/node/244?_x_tr_sl=auto&_x_tr_tl=de&_x_tr_hl=de&_x_tr_pto=wapp as inspiration, but I'd rather make it a general rule, because if an EB resolution is required, the lack of it might be abused for undemocratic purposes.